test/performance/odp_l2fwd.c

```diff
@@ -974,29 +973,29 @@ static int create_pktio(const char *dev, int idx, int num_rx, int num_tx, odp_po
 		uint32_t maxlen_output = pktio_capa.maxlen.max_output ? gbl_args->appl.mtu : 0;
 
 		if (!pktio_capa.set_op.op.maxlen) {
-			ODPH_ERR("Error: modifying interface MTU not supported %s\n", dev);
+			ODPH_ERR("Modifying interface MTU not supported: %s\n", dev);
 			return -1;
 		}
 
 		if (maxlen_input &&
 		    (maxlen_input < pktio_capa.maxlen.min_input ||
 		     maxlen_input > pktio_capa.maxlen.max_input)) {
-			ODPH_ERR("Error: unsupported MTU value %" PRIu32 " for %s "
+			ODPH_ERR("Unsupported MTU value %" PRIu32 " for %s "
 				 "(min %" PRIu32 ", max %" PRIu32 ")\n", maxlen_input, dev,
 				 pktio_capa.maxlen.min_input, pktio_capa.maxlen.max_input);
 			return -1;
 		}
 		if (maxlen_output &&
 		    (maxlen_output < pktio_capa.maxlen.min_output ||
 		     maxlen_output > pktio_capa.maxlen.max_output)) {
-			ODPH_ERR("Error: unsupported MTU value %" PRIu32 " for %s "
+			ODPH_ERR("Unsupported MTU value %" PRIu32 " for %s "
 				 "(min %" PRIu32 ", max %" PRIu32 ")\n", maxlen_output, dev,
 				 pktio_capa.maxlen.min_output, pktio_capa.maxlen.max_output);
 			return -1;
 		}
 
 		if (odp_pktio_maxlen_set(pktio, maxlen_input, maxlen_output)) {
-			ODPH_ERR("Error: setting MTU failed %s\n", dev);
+			ODPH_ERR("Setting MTU failed %s\n", dev);
```

Comment:
":" after "failed" to match the new style.

Reply:
Fixed in v2